guids-generator.html

```diff
@@ -456,17 +482,40 @@
                         />
                     </Grid>
                     <Grid item>
-                        <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
+                      <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
                     </Grid>
                 </Grid>
                 { Object.keys(bhashes).length > 0 &&
-                    <List>
-                        { validatedData.map( item =>
-                            <ListItem className={classes.listItem}>
-                                <ListItemText primary={bhashes[item.index].value} style={{color: bhashes[item.index].isError ? 'red' : 'black'}} />
-                            </ListItem>
-                        )}
-                    </List>
+                    <TableContainer component={Paper}>
+                        <Table className={classes.table} size="small">
+                            <TableHead>
+                              <TableRow>
+                                <StyledTableCell>Health card #</StyledTableCell>
+                                <StyledTableCell>Province</StyledTableCell>
+                                <StyledTableCell>Date of birth</StyledTableCell>
+                                <StyledTableCell>GUID
+                                  <CopyToClipboard text={validatedData.map(item => item.isError ? '' : bhashes[item.index].value).join('\t')} onCopy={() => setCopied(true)}>
+                                    <IconButton title="Copy GGUIDs to clipboard" >
+                                      <Icon>file_copy</Icon>
+                                    </IconButton>
```

Comment:
I would suggest making the button more prominent (e.g. primary color background) and adding text to it (e.g. Copy GUIDs)

Reply:
Also it would be nice if:
- when the user clicks the button and after the content is copied to clipboard, the button would turn green, the icon would be a checkmark, and the label would be "Copied"
- ~3 seconds later the button would revert to its original look, so it's clear for the user they can copy again
- the two modes of the button (or the two different buttons should you choose to implement it that way) should have identical sizes for a consistent look